aarch64_linux/build_aarch64_wheel.py

```diff
@@ -768,7 +773,7 @@ def parse_arguments():
         configure_system(host,
                          compiler=args.compiler,
                          python_version=python_version,
-                         enable_mkldnn=False)
+                         enable_mkldnn=not args.disable_mkldnn)
```

Comment:
This is not needed, as `torch` is not built, but installed from PyPI, i.e. feel free to just omit it
```suggestion
                         )
```